.github/workflows/webview-update.yml

```diff
@@ -0,0 +1,47 @@
+name: Webview Update
+
+on:
+  release:
+    types:
+      - created
```

Comment:
This won't work, because the release will be created by a github action, and github actions may not trigger other github actions. Instead, you should chain this into the release-please workflow as a second job that depends on the release-please job and only runs (with an `if`) in case the release-please job has created a release. For an example of this pattern, check out https://github.com/snakemake/snakemake-interface-executor-plugins/blob/main/.github/workflows/release-please.yml